services/elasticache/pkg/resource/cache_parameter_group/custom_api.go

```diff
@@ -52,9 +53,58 @@ func (rm *resourceManager) customSetOutputDescribeCacheParameters(
 		return err
 	}
 	ko.Status.Parameters = parameters
+	rm.customSetOutputSupplementAPIs(ctx, cacheParameterGroupName, ko)
```

Comment:
Please handle the returned `(bool, error)` from `customSetOutputSupplementAPIs()`

Reply:
Removed `bool` return as it wasn't required.